sql/sql_class.h

```diff
@@ -3550,6 +3550,7 @@ class THD :public MDL_context_owner,
   } binlog_evt_union;
 
 #ifdef WITH_WSREP
+  ulonglong                 wsrep_bin_log_flag_save;
```

Comment:
Can we initialize it to 0?
```suggestion
  ulonglong                 wsrep_bin_log_flag_save{0};
```

Reply:
done